src/workerd/jsg/util.c++

```diff
@@ -677,4 +678,39 @@ v8::Local<v8::String> newExternalTwoByteString(Lock& js, kj::ArrayPtr<const uint
   return check(ExternTwoByteString::createExtern(js.v8Isolate, buf));
 }
 
+// ======================================================================================
+// Node.js Compat
+
+namespace {
+// This list must be kept in sync with the list of builtins from Node.js.
+// It should be unlikely that anything is ever removed from this list, and
+// adding items to it is considered a semver-major change in Node.js.
+static const std::set<kj::StringPtr> NODEJS_BUILTINS{"_http_agent"_kj, "_http_client"_kj,
```

Comment:
if we are only checking for contains, we can use std::array and make this constexpr instead of dynamic allocation. 

if you don't prefer, i recommend making this an unordered_set which will make things faster